infrastructure/cli/commands/helm/history.ts

```diff
@@ -0,0 +1,45 @@
+import {helm, releaseName} from './index.js'
+import {namespace} from '../../lib/cluster.js'
+import {makeCommand} from '../../lib/command.js'
+import {shortDateTime} from '../../lib/misc.js'
+import {EnvironmentOptions} from '../../lib/options.js'
+import {printTable} from '../../lib/table.js'
+
+export const {command, description, builder} = makeCommand(
+  import.meta.url,
+  'List history of releases',
+)
+
+type Release = {
```

Comment:
I see this `Release` type is duplicated here, may be worth moving this type in `helm/index.ts` and import here and in the other places where it's used.

Reply:
the keys are different! maybe i should name these types better/differently…